docs/reference.md

```diff
@@ -711,12 +709,12 @@ time in seconds since the Unix epoch) - and more, depending upon the provider us
 will be returned. Otherwise, the return value will be `null`.
 
 ```js
-var authData = $scope.authObj.$getAuth();
+var firebaseUser = $scope.authObj.$getAuth();
```

Comment:
Hrm, did we change this to $getCurrentUser() in another method? It's pretty strange construct right now, with this revision:  "the firebase user is equal to the auth object dot get auth"